server/public/model/system.go

```diff
@@ -79,6 +79,8 @@ type ServerBusyState struct {
 }
 
 type SupportPacket struct {
```

Comment:
I think it's a good time to add a support packet version. We should start from somewhere.